api/create_distributed_hypertable.md

```diff
@@ -95,4 +95,4 @@ be removed in a future release. To avoid such inconsistency, we do not yet
 recommend using `replication_factor` > 1, and instead rely on physical
 replication of each data node if such fault-tolerance is required.
 
-[create-hypertable]: /timescaledb/:currentVersion:/how-to-guides/hypertables/best-practices
+[create-hypertable]: timescaledb/:currentVersion:/how-to-guides/hypertables/best-practices
```

Comment:
I *think* this may break things, I think it may have to be
```suggestion
[create-hypertable]: /timescaledb/:currentVersion:/how-to-guides/hypertables/best-practices
```

Reply:
@ryanbooz would you agree with this?

Reply:
It would have broken things prior to the impending patch (scheduled for tomorrow) but as a matter of good practice I think starting forward slash should be standard!